lib/archethic/mining/validation_context.ex

```diff
@@ -1137,7 +1154,8 @@ defmodule Archethic.Mining.ValidationContext do
          transaction: tx
        }) do
     case pow do
-      "" ->
+      <<0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0,
```

Comment:
Suggestion: you could probably shorten this as: `<<0::16, 0::256>>`  or even `<<0::272>>` even if the latter might a bit more confusing in terms of understanding.